src/goto-programs/remove_complex.cpp

```diff
@@ -222,12 +229,12 @@ static void remove_complex(exprt &expr)
 
   if(expr.id()==ID_complex_real)
   {
-    assert(expr.operands().size()==1);
+    static_cast<void>(to_complex_real_expr(expr));
```

Comment:
Better use this in the subsequent statement: `expr = complex_member(to_complex_real_expr(expr).op(), ID_real)`